### YamlMime:ManagedReference
items:
- uid: System.Data.Linq.ChangeConflictException
  id: ChangeConflictException
  children:
  - System.Data.Linq.ChangeConflictException.#ctor
  - System.Data.Linq.ChangeConflictException.#ctor(System.String)
  - System.Data.Linq.ChangeConflictException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: ChangeConflictException
  nameWithType: ChangeConflictException
  fullName: System.Data.Linq.ChangeConflictException
  type: Class
  summary: "Fel olvashatja Ha egy frissítés sikertelen lesz, mivel az adatbázis értékek óta frissített az ügyfél utolsó őket."
  remarks: "A következő példa bemutatja, hogyan ehhez a kivételhez használhatók feldolgozási ütközés információt ad vissza."
  example:
  - >-
    [!code-cs[System.Data.Linq.ObjectChangeConflict#1](~/add/codesnippet/csharp/objconflict/program.cs#1)]
     [!code-vb[System.Data.Linq.ObjectChangeConflict#1](~/add/codesnippet/visualbasic/objconflict/module1.vb#1)]
  syntax:
    content: 'public class ChangeConflictException : Exception'
  inheritance:
  - System.Exception
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Data.Linq.ChangeConflictException.#ctor
  id: '#ctor'
  parent: System.Data.Linq.ChangeConflictException
  langs:
  - csharp
  name: ChangeConflictException()
  nameWithType: ChangeConflictException.ChangeConflictException()
  fullName: System.Data.Linq.ChangeConflictException.ChangeConflictException()
  type: Constructor
  assemblies:
  - System.Data.Linq
  namespace: System.Data.Linq
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Data.Linq.ChangeConflictException&quot;> </xref> osztály."
  syntax:
    content: public ChangeConflictException ();
    parameters: []
  overload: System.Data.Linq.ChangeConflictException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.Linq.ChangeConflictException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Data.Linq.ChangeConflictException
  langs:
  - csharp
  name: ChangeConflictException(String)
  nameWithType: ChangeConflictException.ChangeConflictException(String)
  fullName: System.Data.Linq.ChangeConflictException.ChangeConflictException(String)
  type: Constructor
  assemblies:
  - System.Data.Linq
  namespace: System.Data.Linq
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Data.Linq.ChangeConflictException&quot;> </xref> osztályhoz, és adja meg egy üzenetet, amely a kivétel ismertetik."
  syntax:
    content: public ChangeConflictException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Az üzenet ki vannak téve, amikor a kivétel történt."
  overload: System.Data.Linq.ChangeConflictException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.Linq.ChangeConflictException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Data.Linq.ChangeConflictException
  langs:
  - csharp
  name: ChangeConflictException(String,Exception)
  nameWithType: ChangeConflictException.ChangeConflictException(String,Exception)
  fullName: System.Data.Linq.ChangeConflictException.ChangeConflictException(String,Exception)
  type: Constructor
  assemblies:
  - System.Data.Linq
  namespace: System.Data.Linq
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Data.Linq.ChangeConflictException&quot;> </xref> osztály, adja meg egy üzenetet, amely a kivétel azt ismertetik, és adja meg a kivétellel, hogy ezt a kivételt okozott."
  syntax:
    content: public ChangeConflictException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "Az üzenet ki vannak téve, amikor a kivétel történt."
    - id: innerException
      type: System.Exception
      description: "Megadja a kivételt, amely <xref href=&quot;System.Data.Linq.ChangeConflictException&quot;> </xref> eredménye."
  overload: System.Data.Linq.ChangeConflictException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Exception
  isExternal: false
  name: System.Exception
- uid: System.Data.Linq.ChangeConflictException.#ctor
  parent: System.Data.Linq.ChangeConflictException
  isExternal: false
  name: ChangeConflictException()
  nameWithType: ChangeConflictException.ChangeConflictException()
  fullName: System.Data.Linq.ChangeConflictException.ChangeConflictException()
- uid: System.Data.Linq.ChangeConflictException.#ctor(System.String)
  parent: System.Data.Linq.ChangeConflictException
  isExternal: false
  name: ChangeConflictException(String)
  nameWithType: ChangeConflictException.ChangeConflictException(String)
  fullName: System.Data.Linq.ChangeConflictException.ChangeConflictException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Data.Linq.ChangeConflictException.#ctor(System.String,System.Exception)
  parent: System.Data.Linq.ChangeConflictException
  isExternal: false
  name: ChangeConflictException(String,Exception)
  nameWithType: ChangeConflictException.ChangeConflictException(String,Exception)
  fullName: System.Data.Linq.ChangeConflictException.ChangeConflictException(String,Exception)
- uid: System.Data.Linq.ChangeConflictException.#ctor*
  parent: System.Data.Linq.ChangeConflictException
  isExternal: false
  name: ChangeConflictException
  nameWithType: ChangeConflictException.ChangeConflictException
